apps/meteor/server/services/apps-engine/service.ts

```diff
@@ -90,6 +90,18 @@ export class AppsEngineService extends ServiceClassInternal implements IAppsEngi
 
 			// avoid updating the setting if the value is the same,
 			// which caused an infinite loop
+			// and sometimes the settings can be an array
+			// so we need to convert it to JSON stringified to compare it
+
+			if (Array.isArray(oldSetting) && Array.isArray(setting.value)) {
+				if (JSON.stringify(oldSetting) === JSON.stringify(setting.value)) {
```

Comment:
general query, is the insert function always sorted?

Reply:
Jesus man, I posted it first :kek:

Reply:
😂 sorry man didn't see hahahaha

Reply:
consider mine resolved 😂